server/routes/uploadImages.js

```diff
@@ -0,0 +1,33 @@
+
+const aws = require('aws-sdk');
+const express = require('express');
+const router = express.Router();
+const protect = require('../middleware/auth');
+const multer = require('multer');
+const multerS3 = require('multer-s3');
+const { v4: uuidv4 } = require('uuid');
+const config = require('../controllers/utils/awsS3Config');
+
+const s3 = new aws.S3(config);
+
+const uploadToS3 = multer({
+    storage: multerS3({
```

Comment:
seems the same function in `uploadImage.js`. I think it should be the same file, and depend on if it's multi file or single, it should be handled in a more scalable way